doc/reference/configuration/configuration_reference.rst

```diff
@@ -2474,7 +2548,13 @@ The ``process`` defines configuration parameters of the Tarantool process in the
 
 .. confval:: process.work_dir
 
-    A directory where Tarantool working files are stored.
+    **Since:** :doc:`3.0.0 </release/3.0.0>`.
+
+    A directory where database working files will be stored. The server instance
+    switches to ``process.work_dir`` with :manpage:`chdir(2)` after start. Can be
+    relative to the current directory. If not specified, defaults to
+    the current directory. Other directory parameters may be relative to ``process.work_dir``,
+    for example, :ref:`directories for storing snapshots and write-ahead logs <configuration_options_directories>`
```

Comment:
A couple of nits.

> A directory where database working files will be stored.

Not only the database files, but also logs, a PID file, a console Unix socket and other files if an application generates them in the current directory. 'Tarantool working files' seems a good general term if we don't want to go into details here.

> Can be relative to the current directory.

If we agree to follow my proposal above about more precise speaking about relative paths, I would replace it with 'if set as a relative path, it is relative to the current working directory, from where tarantool is started'.

> Other directory parameters may be relative to ``process.work_dir`` <...>

To be precise: directory and file parameters.

If we agree to follow the more precise wording about relative paths, I would say 'other directory and file parameters are interpreted as relative to `process.work_dir` if set as relative paths' or something of this kind.

Don't know, if it is too formal or not.